backend/authentication/factories.py

```diff
@@ -55,6 +91,21 @@ class Meta:
     def verification_partner(
         self, create: bool, extracted: bool, **kwargs: dict[str, Any]
     ) -> None:
+        """
+        Post-generation hook for the `verification_partner` field.
+
+        This workaround is needed because the `verification_partner` field references itself.
+        It does nothing if not in `create` mode.
+
+        Parameters
+        ----------
+        create : bool
+            Whether the object is being created (True) or just built (False).
+        extracted : bool
```

Comment:
Can we have a vertical space between each entry in `Parameters`? It makes it all a bit more readable :)